src/khoj/routers/api.py

```diff
@@ -70,8 +70,35 @@ async def set_content_config_github_data(updated_config: GithubContentConfig):
         except Exception as e:
             return {"status": "error", "message": str(e)}
 
+    @api.post("/delete/config/data/content_type/{content_type}", status_code=200)
```

Comment:
Should we use a `@api.delete` at the existing `/config/data/content_type/{content_type}` endpoint? This would reduce the number of new API endpoints Khoj has to maintain

Reply:
Ack, I'd have to make corresponding updates on Lantern as well to handle the delete request. I'm going to leave it as-is for now just for expedience.